src/test/system/base_system.sol

```diff
@@ -75,13 +75,22 @@ contract BaseSystemTest is TestSetup, Math, DSTest {
         collateralNFT.transferFrom(sender, recipient, tokenId);
     }
 
-
     function issueNFT(address usr) public returns (uint tokenId, bytes32 lookupId) {
         tokenId = collateralNFT.issue(usr);
         lookupId = keccak256(abi.encodePacked(collateralNFT_, tokenId));
         return (tokenId, lookupId);
     }
 
+    function computeCeiling(uint riskGroup, uint nftPrice) public returns (uint) {
+        uint ceilingRatio = nftFeed.ceilingRatio(riskGroup);
```

Comment:
Why are you computing the ceiling instead of getting it directly from the nft feed?

Just for not depending on it in the tests?

Reply:
yes, it is just the ceiling value to expect